### ISSUE TITLE
Bioc checks

### ISSUE THREAD

**Issue by bschilder** [neurogenomics/EpiCompare#87]
EpiCompare is currently not passing Bioc checks, tho im not sure exactly why. Error messages mention about dbplyr but we don't require that as a dep (at least no directly).

 https://bioconductor.org/checkResults/devel/bioc-LATEST/EpiCompare/

![Screenshot 2022-06-06 at 20 40 22](https://user-images.githubusercontent.com/34280215/172235307-02996246-ecf9-40e4-9c0c-3f902117e507.png)

**Comment by bschilder** [comment]
After resolving the issue with pushing upstream #115, @serachoi1230 has pushed to the upstream Bioc 3.16.
It will takes several days for the Bioc servers to recheck and update the build, so we won't see if this has been resolved until after. 
https://www.bioconductor.org/packages/devel/bioc/html/EpiCompare.html

I did notice in the Bioc developer email listserve that some other packages were running into this same issue.

Note, this isn't an issue on the current release version (Bioc 3.15), as all checks are passing there.
https://www.bioconductor.org/packages/release/bioc/html/EpiCompare.html
